packages/lesswrong/components/posts/PostsPage/PostsPageWrapper.tsx

```diff
@@ -39,7 +39,7 @@ const PostsPageWrapper = ({ sequenceId, version, documentId }: {
     if (isMissingDocumentError(error)) {
       return <Error404/>
     } else if (isOperationNotAllowedError(error)) {
-      return <Components.ErrorAccessDenied/>
+      return <Components.ErrorAccessDenied explanation={"This is usually because the post in question has been removed by the author."}/>
```

Comment:
Problem: sometimes we will draft infohazard posts. Then this message is misleading, albeit true. Maybe that's fine. It does say "usually".

Reply:
Hmm, I think usually does the trick here, there are lots of reasons why a post might be inaccessible and I don't think it's worth trying to explain all of them. Going to merge this for "not letting the perfect get in the way of the good" reasons